samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -208,8 +262,8 @@ ov::Tensor get_image_tensor(const std::vector<std::string>& files,
                             const std::pair<std::string, benchmark_app::InputInfo>& inputInfo,
                             std::string* filenames_used = nullptr) {
     auto type = inputInfo.second.type;
-    if (type == ov::element::f32) {
-        return create_tensor_from_image<float>(files,
+    if (type == ov::element::bf16) {
+        return create_tensor_from_image<ngraph::bfloat16>(files,
                                                inputId,
                                                batchSize,
                                                inputInfo.second,
```

Comment:
There are a few types miisng - namely `i4`, `u1`, `u4` - what should I do with them?

Reply:
If it is easy to add them, do so. If not, you can leave them.

Reply:
I guess, @nkogteva is right. I didn't think that types like `i4`, `u1`, `u4` aren't actually applicable as inputs

Reply:
Great to hear, I was a bit worried about their stability to be honest, although my pipeline seemed to work just fine. Regardless, I removed the useless types!